### DISCUSSION TITLE
Energy dashboard sandbox please

### DISCUSSION THREAD

**Discussion by LKapie** [home-assistant/frontend#14294] [Dashboard]
The energy dashboard is a very nice feature and by using it I was wondering about several "what if" scenario's like:
- What if I buy solar panels?
- What if I add a battery to my system?
- What if energy prices rise?
- What if I replace my current gas boiler with an electric heat pump?

It's not to difficult to add simulation devices like solar panel simulation etc.
But at the moment there is no "sandbox" energy dashboard where I can simulate scenario's based on the actual energy usage.
It seems not to difficult to add it: the ability to add more energy dashboards with there own cost calculations.

Nice to haves:
- some helpers for the simulation devices (solar panels, simulate electric consumption instead of gas, battery ...)
- compare with other/real energy dashboard/periods

**Comment by wrfz** [comment]
I also really miss this feature.
I would like to use my hourly data (PV production and consumption data) to simulate how much my electricity costs would have been if I had a storage system.
As an HA user, I would like to store a few settings in the energy dashboard (e.g. simulated capacity, charging/discharging depth, efficiency, ...) and then activate or deactivate the storage with a checkbox and thus compare the consumption.


Translated with www.DeepL.com/Translator (free version)
